bee-network/src/config.rs

```diff
@@ -41,6 +41,14 @@ pub enum Error {
     /// The domain was unresolvable.
     #[error("Domain name '{}' couldn't be resolved to an IP address", .0)]
     UnresolvableDomain(String),
+
+    ///
```

Comment:
:trollface:

Reply:
Interesting that `clippy` doesn't catch those things. Would be easy, right? :smile: